tests/unit/formats/bbcode.parser.js

```diff
@@ -872,11 +872,13 @@ QUnit.test('Code', function (assert) {
 		'Normal'
 	);
 
+	/* TODO: Find out why this assertion fails
 	assert.htmlEqual(
 		this.parser.toHTML('[code]Testing [b]test[/b][/code]'),
 		'<code>Testing [b]test[/b]<br /></code>',
 		'Normal'
 	);
+	*/
```

Comment:
Strangely enough, this test fails where `[b]` is converted to `<strong>`, even though similar code behaves on both Firefox and Chrome or me.

			var parser = sceditor.instance(document.getElementById('example'));
			console.log(parser.fromBBCode(
				'[code]Testing [b]test[/b][/code]'
			));

i''m guessing this may be why `handleTags()` differentiated between block and inline.

Also, the tests are a pain to debug because not only the line numbers are messed up (yay for monolithic files) but `console.log()` doesn't work.

Reply:
ok so  I reverted that one and surprisingly the assertion passes here. Dunno why. Not gonna look into it because this test suite is a royal pain for me to debug.

I wonder if it's because I use Windows.

Reply:
And now after adding the latest gamut of tests, the problematic one passes on my end. Amusing.

Seems like a consequence of global state.

Reply:
That's really strange, I might try an investigate the cause of this as it looks like a bug somewhere in the code.

Reply:
My workaround is https://github.com/samclarke/SCEditor/pull/821/commits/81a56bbadc5c7ab2db8f512de1600dbc3cea4c5f

Seems to be global state. I'm leaving it be since it works.

Reply:
Oh I see, it's a bug with one of the tests replacing the test parser instance :+1: